definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -48,5 +48,49 @@
     "Label": "Reason for refusal",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceApplicationPayment",
+    "Label": "How will payment be made?",
+    "HintText": "How will payment be made?",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "ServiceApplicationPaymentTypeEnum",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "03/08/2020",
```

Comment:
I believe we already have a PBA Number field. Are you really supposed to create a new one?

Reply:
Not as far as i could see from the code. And yes, i believe so

Reply:
I think this is the one, @jacmurphy: https://github.com/hmcts/div-ccd-definitions/blob/div6477-prodRelease/definitions/divorce/json/CaseField/CaseField.json#L1756

Reply:
Can you please check with Jeremy?

Reply:
Waiting on response